src/birthder.js

```diff
@@ -208,26 +208,45 @@
         return QUOTES[(Math.random() * QUOTES.length) >> 0];
     }
 
+    /**
+     * Form a reminder message
+     *
+     * @returns {string}
+     */
+     function formReminderMessage(users, targetDay, amountOfTime) {
+        let userNames, userNamesString, toBe, when, message;
```

Comment:
Don't inherit a bad style of coding. Instead of this, declare each variable separately. Moreover, it's better to make them constants.